docs/doc/14-sql-commands/10-dml/dml-copy-into-table.md

```diff
@@ -332,81 +338,114 @@ CONNECTION = (endpoint_url = 'https://<your-ipfs-gateway>')
 FILE_FORMAT = (type = CSV field_delimiter = ',' record_delimiter = '\n' skip_header = 1);
 ```
 </TabItem>
-
 </Tabs>
 
-### 4. Loading Data with Pattern Matching
+### Example 3: Loading Compressed Data
 
-This example uses pattern matching to only load from CSV files containing `sales` in their names:
+This example loads 10 rows of data from a GZIP-compressed CSV file on Amazon S3 into Databend:
 
 ```sql
 COPY INTO mytable
-FROM 's3://mybucket/'
-PATTERN = '.*sales.*[.]csv'
-FILE_FORMAT = (type = CSV field_delimiter = ',' record_delimiter = '\n' skip_header = 1);
+FROM 's3://mybucket/data.csv.gz'
+CONNECTION = (
+    ENDPOINT_URL = 'https://<endpoint-URL>',
+    ACCESS_KEY_ID = '<your-access-key-ID>',
+    SECRET_ACCESS_KEY = '<your-secret-access-key>'
+)
+FILE_FORMAT = (type = CSV field_delimiter = ',' record_delimiter = '\n' skip_header = 1 compression = AUTO)
+SIZE_LIMIT = 10;
```

Comment:
the SIZE_LIMIT is wrongly implemented now, please  remove `SIZE_LIMIT = 10;` from example